Code/GraphMol/RGroupDecomposition/RGroupDecompData.h

```diff
@@ -52,20 +52,21 @@ struct RGroupDecompData {
         newCoreLabel(EMPTY_CORE_LABEL),
```

Comment:
I don't think that you need to default initialize `cores` and `newCores`, and if you used 

    int newCoreLabel = EMPTY_CORE_LABEL;

in the definition, you wouldn't need to define that, either. (see CPP core guidelines, here: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c48-prefer-in-class-initializers-to-member-initializers-in-constructors-for-constant-initializers)

Reply:
Good point, I had noticed the initializers in passing but then I forgot to remove them.